packages/amplify-graphql-api-construct-tests/src/__tests__/sql-models-1.test.ts

```diff
@@ -41,37 +41,29 @@ describe('CDK GraphQL Transformer deployments with SQL datasources', () => {
     await databaseController.cleanupDatabase();
   });
 
-  beforeEach(async () => {
-    projRoot = await createNewProjectDir(projFolderName);
-  });
-
-  afterEach(async () => {
-    try {
-      await cdkDestroy(projRoot, '--all');
-    } catch (err) {
-      console.log(`Error invoking 'cdk destroy': ${err}`);
-    }
-
-    deleteProjectDir(projRoot);
-  });
-
-  test('creates a GraphQL API from SQL-based models with Secrets Manager Credential Store default encryption key', async () => {
-    await testGraphQLAPI(constructTestOptions('secretsManager'));
-  });
-
-  test('creates a GraphQL API from SQL-based models with Secrets Manager Credential Store custom encryption key', async () => {
-    await testGraphQLAPI(constructTestOptions('secretsManagerCustomKey'));
-  });
-
-  test('creates a GraphQL API from SQL-based models with Secrets Manager Credential Store default encryption key', async () => {
-    await testGraphQLAPI(constructTestOptions('secretsManagerManagedSecret'));
-  });
-
   const constructTestOptions = (connectionConfigName: string) => ({
-    projRoot,
+    projFolderName,
     region,
     connectionConfigName,
     dbController: databaseController,
     resourceNames,
   });
+
+  testGraphQLAPI(
+    constructTestOptions('secretsManager'),
+    'creates a GraphQL API from SQL-based models with Secrets Manager Credential Store default encryption key',
+    ImportedRDSType.MYSQL,
+  );
+
+  testGraphQLAPI(
+    constructTestOptions('secretsManager'),
```

Comment:
Does this need to be `secretsManagerCustomKey`, since otherwise it's pretty similar to the above test.